packages/web/src/components/gcds-card/gcds-card.tsx

```diff
@@ -143,42 +167,44 @@ export class GcdsCard {
       taggedAttr['aria-describedby'] = 'gcds-badge';
     }
 
-    return (
-      <Host>
-        <div class="gcds-card">
-          {imgSrc && (
-            <img
-              src={imgSrc}
-              alt={imgAlt ? imgAlt : ''}
-              class="gcds-card__image"
-            />
-          )}
-          {(badge && !badgeError) && (
-            <gcds-text
-              id="gcds-badge"
-              class="gcds-badge"
-              text-role="light"
-              margin-bottom="0"
-              size="caption"
-            >
-              <strong>
-                <gcds-sr-only>{i18n[lang].tagged}</gcds-sr-only>
-                {badge}
-              </strong>
-            </gcds-text>
-          )}
-          {Element != 'a' ? (
-            <Element class="gcds-card__title" {...taggedAttr}>
-              <gcds-link href={href}>{cardTitle}</gcds-link>
-            </Element>
-          ) : (
-            <gcds-link href={href} class="gcds-card__title" {...taggedAttr}>
-              {cardTitle}
-            </gcds-link>
-          )}
-          <div class="gcds-card__description">{renderDescription}</div>
-        </div>
-      </Host>
-    );
+    if ((!error.includes('href') && !error.includes('cardTitle'))) {
```

Comment:
Is this possible if we add a required props validation?
`if(!this.validateRequiredProps()){`